objects/ohitbox/Step_2.gml

```diff
@@ -67,7 +67,8 @@ if (collisionCheck)
 				hasHit = true;
 			} else
 			// Grabbing
-			if attackProperty.attackType[hitboxID] == eAttackType.GRAB && collision_list[| i].owner.hitstun < 1 && collision_list[| i].owner.blockstun < 1 && collision_list[| i].owner.grounded == true && collision_list[| i].owner.state != eState.THROW_TECH && collision_list[| i].owner.state != eState.JUMPSQUAT{
+			if (attackProperty.attackType[hitboxID] == eAttackType.GRAB && collision_list[| i].owner.hitstun < 1 && collision_list[| i].owner.blockstun < 1 && collision_list[| i].owner.grounded == true && collision_list[| i].owner.state != eState.THROW_TECH && collision_list[| i].owner.state != eState.JUMPSQUAT && collision_list[| i].owner.isThrowable)

```

Comment:
This if statement is getting really long, and there's a ```boolean == true``` in there... let's reformat it like this to be more readable (you should be able to basically copy paste this... spacing might be off but not by much):

```
if (attackProperty.attackType[hitboxID] == eAttackType.GRAB &&
	collision_list[| i].owner.hitstun < 1 &&
	collision_list[| i].owner.blockstun < 1 &&
	collision_list[| i].owner.grounded &&
	collision_list[| i].owner.state != eState.THROW_TECH &&
	collision_list[| i].owner.state != eState.JUMPSQUAT &&
	collision_list[| i].owner.isThrowable)
```